simbad/command_line/simbad_full.py

```diff
@@ -61,7 +61,7 @@ def main():
         raise OSError("amore executable not found")
 
     gui = simbad.util.pyrvapi_results.SimbadOutput(
-        args.rvapi_document, args.webserver_uri, args.display_gui, log_file, args.work_dir, ccp4i2_xml=args.ccp4i2_xml
```

Comment:
We need this, in case this becomes/is a script invoked through the GUI. It's probably best to leave it as a keyword argument though.